### ISSUE TITLE
ExpresseLRS-Configurator setup flagged by Windows Security as containing malware (Trojan:Win32/Emali.Ac!l)

### ISSUE THREAD

**Issue by CdtSylvestre** [ExpressLRS/ExpressLRS-Configurator#30]
## Expected Behavior

Installation of the configurator on Windows 10 using the ExpressLRS-Configurator-Setup-0.5.3.exe

## Current Behavior

Flagged as malware (trojan:Win32/Emali.Ac!l) by Microsoft Defender (default Windows Security), file quarantined, cannot install.

![ExpressLRS-configurator-virus](https://user-images.githubusercontent.com/53912200/124270450-82911900-db3c-11eb-8f00-283cd414ba89.png)

## Steps to Reproduce
Downloaded ExpressLRS-Configurator-Setup-0.5.3.exe from Releases page
Double-clicked on the *.exe to start the setup
Windows Defender prevents the configurator setup to launch, indicates that it contains a trojan (see screenshot). 

Tried previous version (ExpressLRS-Configurator-Setup-0.5.2.exe), same issue.

**Please note**: if a third-party security suite/antivirus is installed on the PC, Microsoft Defender is partly or fully disabled and the issue will not occur.
 

## Possible Solution (Not obligatory)

Most likely a false positive with Microsoft Defender. The file is clean according to Avast, for instance. Annoying anyway as Defender is probably the primary security solution by default for most Windows 10 users. Tested on three different computers, on separate networks. If the cause of this flagging as a severe threat by Microsoft cannot be identified and resolved and the file is confirmed clean by the devs, a warning about this false positive on the Releases page would perhaps be useful.  

## Context

Building confidence in ExpressLRS, a very nice RC system. Installed on an old laptop instead, after disabling Windows Security, very pleased with the software/hardware solution, great work ! It would be a shame for this issue to drive potential users away.

## Your Environment

- Operating System and version: Windows 10 build 19043.1052, Defender virus and threat definitions up to date as of July 2, 2021.
- TX hardware: Happymodel ES24TX (not relevant for this issue)
- RX hardware: Happymodel EP1 (ditto)

**Comment by jurgelenas** [comment]
Hello @CdtSylvestre,

Thank you for reporting this.

I am confident that this is just a false positive. EdgeTx Flashter had exactly the same [issue ](https://github.com/EdgeTX/flasher/issues/13) and it was resolved by submitting their exec to Microsoft. I have done the same.

Will close the issue once Windows Defender database is updated. I just hope that this process will not be required with each release.

**Comment by jurgelenas** [comment]
My report was processed my Microsoft. False positive report was removed. Can you please check again @CdtSylvestre ?

**Comment by CdtSylvestre** [comment]
Thanks for your work, Julius!

[ExpressLRS-Configurator-Setup-0.5.3.exe](https://github.com/ExpressLRS/ExpressLRS-Configurator/releases/download/v0.5.3/ExpressLRS-Configurator-Setup-0.5.3.exe) now launches and installs the configurator without any issue.

Downloading and attempting to launch the newly released [ExpressLRS-Configurator-Setup-0.5.4.exe](https://github.com/ExpressLRS/ExpressLRS-Configurator/releases/download/v0.5.4/ExpressLRS-Configurator-Setup-0.5.4.exe) _does_ trigger a warning ("unrecognized app", see screenshot) _but_ does not flag the *.exe as containing any particular threat (unlike what it did for 0.5.3 yesterday). The user is then given the choice to run the installer anyway. It installs without any further warning and works as it should.

![ExpressLRS configurator 0 5 4-unrecognized](https://user-images.githubusercontent.com/53912200/124363933-94f37b80-dc3e-11eb-928c-285e4c62308f.jpg)

In short, Windows Security/Microsoft Defender now knows that 0.5.3 is safe. It does not (yet?) know about 0.5.4 but does not flag it as malware either, it just leaves the responsibility to the user to install or not.

**Comment by jurgelenas** [comment]
Awesome. Works as expected now.

Thanks again @CdtSylvestre for your report.

Closing the issue now.
